dds_web/api/api_s3_connector.py

```diff
@@ -9,6 +9,7 @@
 import traceback
 import pathlib
 import json
+import gc
```

Comment:
Not needed. I’m considering to clean such things up some day.

Reply:
There are a bunch of tools to lint / fix unused imports..